File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int index1 = 0;
        int index2 = 0;
        int neededNum;
        int indexNeededNum;
        for(int i = 0; i < nums.Length; ++i){
            neededNum = target - nums[i];
            indexNeededNum = Array.IndexOf(nums, neededNum);
            if(indexNeededNum != i && indexNeededNum >= 0){
                index1 = i;
                index2 = indexNeededNum;
                break;
            }
        }
        return new int[]{index1, index2};
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}
        for i in range(len(nums)):
            if nums[i] in hashMap:
                return [i, hashMap[nums[i]]]
            neededNum = target - nums[i]
            hashMap[neededNum] = i